pkg/datarepair/irreparable/irreparable.go

```diff
@@ -15,8 +15,8 @@ type DB interface {
 	IncrementRepairAttempts(ctx context.Context, segmentInfo *pb.IrreparableSegment) error
 	// Get returns irreparable segment info based on segmentPath.
 	Get(ctx context.Context, segmentPath []byte) (*pb.IrreparableSegment, error)
-	// GetLimited number of segments from offset
-	GetLimited(ctx context.Context, limit int, offset int64) ([]*pb.IrreparableSegment, error)
+	// GetLimited returns a list of irreparable segment info starting after the last segment info we retrieved
+	GetLimited(ctx context.Context, limit int, lastSeenSegmentPath []byte) ([]*pb.IrreparableSegment, error)
```

Comment:
Usually this is called `cursor` in our database code.

Reply:
good point.